orc8r/lib/go/metrics/metrics_export.go

```diff
@@ -24,6 +24,7 @@ import (
 const (
 	NetworkLabelName   = "networkID"
 	GatewayLabelName   = "gatewayID"
+	EnodebLabelName    = "enodebID"
 	CloudHostLabelName = "cloudHost"
 )
 
```

Comment:
Notes for GetMetrics:
1. Remove the superfluous emtpy line at the start of the fn
2. Use errors.Wrap from https://godoc.org/github.com/pkg/errors
3. timeStamp -> timestamp (it's one word)

Reply:
I haven't modified this file actually. Just added a constant here. Will avoid this this cleanup for now to avoid confusing the reader.